src/Common/src/System/Security/Cryptography/DSACng.ImportExport.cs

```diff
@@ -321,7 +321,7 @@ public override DSAParameters ExportParameters(bool includePrivateParameters)
                             dsaParams.Seed = null;
                         }
 
-                        Debug.Assert(offset == 0, "offset == 0");
+                        Debug.Assert(offset == dsaBlob.Length, "offset == dsaBlob.Length");
```

Comment:
Makes sense. Will do. May or may not help; I suppose if the offset is crazy big that could indicate bad data